setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 setup(
-    setup_requires=['pbr'],
+    setup_requires=['pbr', 'setuptools'],
```

Comment:
This is deprecated and must be in `pyproject.toml`'s `[build-system]`.

Reply:
Done.